project/src/main/java/com/google/sps/servlets/CreateProductServlet.java

```diff
@@ -138,8 +138,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     }
     
     
-//TODO:Phillips Product Search functions 
-//     ServletsLibrary.createProduct(projectId, computeRegion, productId, productDisplayName, productCategory);
-//     ServletsLibrary.addProductToProductSet(projectId, computeRegion, productId, setId);
+    // TODO:Phillips Product Search functions 
+    // Functions to create product and add to a product set in the product search database
+    ProductSearchLibrary.createProduct(productId, productDisplayName, productCategory);
```

Comment:
You don't necessarily have to change anything right now, but I wanted to point out that this doPost function is very long now and it would be helpful to break related chunks out into helper functions.

Example:

- one to add to a product set
- one to generate labels
- one to add to datastore
- one to do the product search things